docs/comparisons/opa.md

```diff
@@ -0,0 +1,204 @@
+| Policy enforcement modes                              | deny, warn                  | deny, warn                       |
+| Deployment mode [[6]](#deployment-mode)               | single evaluation server    | multiple evaluation servers      |
+| Background checks [[7]](#background-checks)           | ✅                          | ✅                               |
+
+## Types of policies
+
+Both OPA Gatekeeper and Kubernetes can write validation and mutation policies.
+
+These policies can target any kind of Kubernetes resource, including Custom Resources.
+
+## Writing policies
+
+OPA Gatekeeper policies are written using [Rego](https://www.openpolicyagent.org/docs/latest/#rego).
+Rego is a query language created by the Open Policy Agent project.
+
+:::info
+Rego can be used only to write validating policies. The policies performing mutation do not
+use Rego, but instead use some ad-hoc rules defined via YAML (see [here](https://open-policy-agent.github.io/gatekeeper/website/docs/mutation)).
+:::
+
+Kubewarden allows policies to be written using different styles. Policy authors
+can use both traditional programming languages (like Go, Rust and others) or Rego.
+Kubewarden's validating and mutating policies are written in the same way.
+
+:::caution
+Rego is used also by the [kube-mgmt](https://github.com/open-policy-agent/kube-mgmt)
+open source project, which is part of the Open Policy Agent project.
+
+Despite using the same language, policies written for kube-mgmt are
+not compatible with OPA Gatekeeper and vice versa.
+
+Kubewarden can use Rego policies that have been written for both Open Policy Agent and
+for OPA Gatekeeper. More information can be found [here](https://docs.kubewarden.io/writing-policies/rego/intro-rego).
+:::
+
+## Context aware
+
+Sometimes a policy might need data about the current state of the cluster in order to take a
+validation/mutation decision. For example, a policy validating Ingress resources might
+need to know what are the other Ingress resources already defined inside of the cluster
+to ensure no clash might happen.
+This kind of policies are called "context aware".
+
+Both OPA Gatekeeper and Kubewarden support this type of policies.
+
+:::caution
+When deploying OPA Gatekeeper, a Kubernetes administrator decides which type of
+cluster data should be made available to the policies at evaluation time.
+
+It's important to highlight how this data is then accessible by all the policies
+deployed.
+
+For example, if an OPA Gatekeeper policy requires access to Kubernetes Secrets,
+all the other policies being deployed will be able to read this data as well.
+:::
+
+On the other hand, Kubewarden provides a granular access to cluster resources.
+Each policy has access only to the resources that the Kubernetes administrator
+specified. Attempting to read unauthorized data is immediately blocked and
+reported to the cluster administrators.
```

Comment:
I would say to remove the "caution" callout. For me, callouts try to catch my attention to something in the text. around. But here it is the actual comparision with Kubewarden. Therefore, I would remove it.

Reply:
I proposed the change